File: programmers/level1/부족한금액계산하기/java/20221005/test/SolutionTest.java
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class SolutionTest {

    @Test
    void totalPrice() {
        Solution solution = new Solution();

        assertEquals(30, solution.computeTotalPrice(3, 4));
    }

    @Test
    void checkMoney() {
        Solution solution = new Solution();

        assertEquals(10, solution.checkMoney(20, 30));
    }

    @Test
    void enoughMoney() {
        Solution solution = new Solution();

        assertEquals(0, solution.checkMoney(40, 30));
        assertEquals(0, solution.checkMoney(30, 30));
    }
}

File: programmers/level1/시저암호/javascript/20221021/solution.test.js
import solution from './solution';

test('시저 암호를 구해보자', () => {
  expect(solution('AB', 1)).toBe('BC');
  expect(solution('z', 1)).toBe('a');
  expect(solution('a B z', 4)).toBe('e F d');
});
